src/test/java/Homework17.java

```diff
@@ -0,0 +1,61 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongtoPlaylist() throws InterruptedException {
+        url();
+        inputEmail("guyvanalex4@gmail.com");
+        inputPassword("teststudent!1");
+        clickSubmit();
+        allSongs();
+        songName();
+        addTo();
+        myPlaylist();
+        Assert.assertTrue(getSuccessPopUp().isDisplayed());
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210974871-a54cac24-af6a-42ac-bfbf-7c3ab9678d98.png)

![image](https://user-images.githubusercontent.com/104761482/210974949-e17cd25e-0047-40a0-a6e5-25a38f13c64a.png)